testsuite/MDAnalysisTests/lib/test_distances.py

```diff
@@ -251,50 +283,70 @@ def test_self_capped_distance(self, npoints, box, method, min_cutoff,
+        "box",
+        (
+            None,
+            np.array([1, 1, 1, 90, 90, 90], dtype=np.float32),
+            np.array([1, 1, 1, 60, 75, 80], dtype=np.float32),
+        ),
+    )
+    @pytest.mark.parametrize(
+        "npoints,cutoff,meth",
+        [
+            (1, 0.02, "_bruteforce_capped_self"),
+            (1, 0.2, "_bruteforce_capped_self"),
+            (600, 0.02, "_pkdtree_capped_self"),
+            (600, 0.2, "_nsgrid_capped_self"),
+        ],
+    )
     def test_method_selfselection(self, box, npoints, cutoff, meth):
         np.random.seed(90003)
-        points = (np.random.uniform(low=0, high=1.0,
-                  size=(npoints, 3))).astype(np.float32)
+        points = (
+            np.random.uniform(low=0, high=1.0, size=(npoints, 3))
+        ).astype(np.float32)
         method = distances._determine_method_self(points, cutoff, box=box)
         assert_equal(method.__name__, meth)
 
-    @pytest.mark.parametrize('box', (None,
-                                     np.array([1, 1, 1,  90, 90, 90],
-                                              dtype=np.float32),
-                                     np.array([1, 1, 1, 60, 75, 80],
-                                              dtype=np.float32)))
-    @pytest.mark.parametrize('npoints,cutoff,meth',
-                             [(1, 0.02, '_bruteforce_capped'),
-                              (1, 0.2, '_bruteforce_capped'),
-                              (200, 0.02, '_nsgrid_capped'),
-                              (200, 0.35, '_bruteforce_capped'),
-                              (10000, 0.35, '_nsgrid_capped')])
+    @pytest.mark.parametrize(
+        "box",
+        (
+            None,
+            np.array([1, 1, 1, 90, 90, 90], dtype=np.float32),
+            np.array([1, 1, 1, 60, 75, 80], dtype=np.float32),
+        ),
+    )
+    @pytest.mark.parametrize(
+        "npoints,cutoff,meth",
+        [
+            (1, 0.02, "_bruteforce_capped"),
+            (1, 0.2, "_bruteforce_capped"),
+            (200, 0.02, "_nsgrid_capped"),
+            (200, 0.35, "_bruteforce_capped"),
+            (10000, 0.35, "_nsgrid_capped"),
+        ],
+    )
     def test_method_selection(self, box, npoints, cutoff, meth):
         np.random.seed(90003)
-        points = (np.random.uniform(low=0, high=1.0,
-                  size=(npoints, 3)).astype(np.float32))
+        points = np.random.uniform(low=0, high=1.0, size=(npoints, 3)).astype(
+            np.float32
+        )
         method = distances._determine_method(points, points, cutoff, box=box)
         assert_equal(method.__name__, meth)
 
 
 @pytest.fixture()
 def ref_system():
-    box = np.array([1., 1., 2., 90., 90., 90], dtype=np.float32)
+    box = np.array([1.0, 1.0, 2.0, 90.0, 90.0, 90], dtype=np.float32)
     points = np.array(
         [
-            [0, 0, 0], [1, 1, 2], [1, 0, 2],  # identical under PBC
+            [0, 0, 0],
+            [1, 1, 2],
+            [1, 0, 2],  # identical under PBC
```

Comment:
This comment applies to all the three above. I'd suggest to keep the old formatting here.